Assets/Scripts/UI/MainMenuScript.cs

```diff
@@ -0,0 +1,19 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace UI
+{
+    public class MainMenuScript : MonoBehaviour
```

Comment:
Why is this script a MonoBehaviour, if you don't use any of MonoBehaviour features ?